labs/multiverse/src/main.rs

```diff
@@ -41,6 +42,19 @@ const ALT_ROW_COLOR: Color = tailwind::SLATE.c900;
 const SELECTED_STYLE_FG: Color = tailwind::BLUE.c300;
 const TEXT_COLOR: Color = tailwind::SLATE.c200;
 
+#[derive(Debug, Parser)]
+struct Cli {
+    /// The homeserver the client should connect to.
+    server_name: OwnedServerName,
+    /// The path where session specific data should be stored.
```

Comment:
ubernit: can you add a blank line between the two please?

Reply:
Did so, to fix the typo I had to reword the commit so it's squashed and force pushed already.